[2023-09-19 07:37:10,561] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T00:07:00+00:00 [queued]>
[2023-09-19 07:37:10,886] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T00:07:00+00:00 [queued]>
[2023-09-19 07:37:10,887] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:37:10,887] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2023-09-19 07:37:10,887] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:37:11,151] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2023-09-19T00:07:00+00:00
[2023-09-19 07:37:11,214] {standard_task_runner.py:52} INFO - Started process 360 to run task
[2023-09-19 07:37:11,288] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', '2023-09-19T00:07:00+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/weather_data_pipeline.py', '--cfg-path', '/tmp/tmpk8f9yc_2', '--error-file', '/tmp/tmp6nfli957']
[2023-09-19 07:37:11,288] {standard_task_runner.py:77} INFO - Job 9: Subtask fetch_weather_data
[2023-09-19 07:37:12,182] {logging_mixin.py:104} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T00:07:00+00:00 [running]> on host 92beb7708765
[2023-09-19 07:37:13,163] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data_pipeline
AIRFLOW_CTX_TASK_ID=fetch_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-09-19T00:07:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-19T00:07:00+00:00
[2023-09-19 07:37:14,795] {python.py:151} INFO - Done. Returned value was: True
[2023-09-19 07:37:15,194] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, execution_date=20230919T000700, start_date=20230919T073710, end_date=20230919T073715
[2023-09-19 07:37:15,582] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-19 07:37:15,696] {local_task_job.py:151} INFO - Task exited with return code 0
